### PR TITLE
chore: add prometheus metrics endpoint

### PR DESCRIPTION
**Pull request by luislhl**
### Aceptance Criteria
- There should be a new `/metrics` endpoint in API Gateway
- The new endpoint should return the metric `wallet_service:best_block_height` by making a query in the database

### Questions
- Should we have authentication in this endpoint?

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -63,6 +63,7 @@ provider:
     DB_NAME: ${env:DB_NAME}
     DB_USER: ${env:DB_USER}
     DB_PASS: ${env:DB_PASS}
+    ENVIRONMENT: ${opt:stage, 'dev'}
```
Comment (andreabadesso) [inline serverless.yml]: Use `STAGE`? We are already using `STAGE` on `loadWallet`

**Review by andreabadesso — APPROVED** [review]

File: serverless.yml (hunk 1 above)
Comment (luislhl) [reply serverless.yml]: Will change that

**Review by andreabadesso — APPROVED** [review]

**Comment by pedroferreira1** [comment]
> Should we have authentication in this endpoint?

Maybe not an auth but at least a rate limit or cache to prevent someone from abusing it and calling this lambda a billion times

**Review by pedroferreira1 — APPROVED** [review]

**Comment by luislhl** [comment]
> Maybe not an auth but at least a rate limit or cache to prevent someone from abusing it and calling this lambda a billion times

Rate limit makes sense. We have an issue to make sure all endpoints have one, so I'll let this improvement to this issue.

**Pull request merged**